devito/ops/transformer.py

```diff
@@ -202,22 +204,56 @@ def create_ops_dat(f, name_to_ops_dat, block):
                       ops_decl_dat=ops_decl_dat)
 
 
-def create_ops_fetch(f, name_to_ops_dat, time_upper_bound):
+def initialize_memspace():
+    memspace = OpsMemSpace(namespace['ops_memspace_name'])
+    host_macro = Macro('OPS_HOST')
+
+    return Expression(ClusterizedEq(Eq(memspace, host_macro)))
+
+
+def create_ops_memory_fetch(f, name_to_ops_dat, par_to_ops_stencil, accessibles_info,
+                            memspace):
+    """Create memory fetch calls for a given variable.
+
+    To get the data back to Devito from the device memory it is necessary to call the
+    OPS API method: `ops_dat_get_raw_pointer`.
+
+    Parameters
+    ----------
+    f : Function or TimeFunction
+        Devito object that was transfered into the device memory.
+    name_to_ops_dat : dict of {str : :class:`devito.ops.types.OpsDat`}
+        Given a variable name, get the associated OpsDat object.
+    par_to_ops_stencil : dict of {str : :class:`devito.ops.types.OpsStencil`}
+        Link an OpsAccessible name to its OpsStencil variable.
+    accessibles_info : dict of {str : :namedTuple:`devito.ops.types.Accessible_info`}
+        Contains the time variable used.
+    memspace : :class:`devito.ops.types.OpsMemSpace`
+        Specify which memory space should the data be transfered into.
+
+    Returns
+    ------
+    :class:`devito.ir.iet.nodes.Call`
```

Comment:
no need for `:class:` (remember? we've eliminated them all, thanks to @btrb too)

you should remove all of them

Reply:
Sure.